apps/web/src/lang/fr.json

```diff
@@ -8136,20 +8136,182 @@
+    "description": "Message on education requirements card on the application review page."
+  },
+  "O83lYZ": {
+    "defaultMessage": "Cette section résume votre CV tel que le verront les responsables du recrutement pour ces postes.",
+    "description": "Blurb for resume section of the application review page."
+  },
+  "Sd8883": {
+    "defaultMessage": "Vous avez répondu aux questions suivantes :",
+    "description": "Message in screening questions section of the application review page."
+  },
+  "T8eOrg": {
+    "defaultMessage": "Votre nom complet",
+    "description": "Label displayed for signature input in sign and submit section of application review page."
+  },
+  "PDbqFR": {
+    "defaultMessage": "\"J’ai relu tout le contenu de ma candidature\"",
+    "description": "Review list item for sign and submit section of application review page."
+  },
+  "V9lxDE": {
+    "defaultMessage": "Il semble que vous n’ayez pas encore répondu aux questions de sélection.",
+    "description": "Null state message in screening questions section of the application review page."
+  },
+  "bO9PB4": {
+    "defaultMessage": "J’envoie ma candidature"
+  },
+  "8BR20I": {
+    "defaultMessage": "Nous vous encourageons à tenir votre profil à jour, car c’est sa version actualisée qui servira lors des étapes à venir du processus d’embauche.",
+    "description": "List item note 3 for the application review page."
+  },
+  "pH8wF2": {
+    "defaultMessage": "Bravo, c’est réussi! En signant, vous confirmez ce qui suit :",
+    "description": "Instructions for sign and submit section of application review page."
+  },
+  "Pasf+O": {
+    "defaultMessage": "Les mises à jour de votre profil après l’envoi de votre candidature ne se refléteront pas dans votre dossier de candidature.",
+    "description": "List item note 2 for the application review page."
+  },
+  "fhgZRX": {
+    "defaultMessage": "Je signe et j’envoie ma candidature",
+    "description": "Heading for sign and submit section of application review page."
+  },
+  "h3Cr80": {
+    "defaultMessage": "Il semble que vous n’ayez pas encore ajouté d’expérience professionnelle à votre CV.",
+    "description": "Null state message for experiences section of the application review page."
+  },
+  "jV/FG1": {
+    "defaultMessage": "Modifiez la formation requise",
+    "description": "Edit link text for education requirements section of the application review page."
+  },
+  "jX2LG0": {
+    "defaultMessage": "Compétences requises",
+    "description": "Heading for skill requirements section of the application review page."
+  },
+  "mf8++l": {
+    "defaultMessage": "Il semble que vous n’ayez pas encore sélectionné de formation requise.",
+    "description": "Null state message for education requirement section of the application review page."
+  },
+  "mu6uvE": {
+    "defaultMessage": "Modifiez cette section"
```

Comment:
No description for i18n?

Reply:
I'm not sure, I did not add this string 😅